examples/gkp_phase_gate.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A comparison of the Wigner function for a finite energy GKP |+i> state
+to the output of a GKP qubit phase gate applied to a finite energy |+> state.
+The qubit phase gate is applied using measurement-based squeezing."""
+
+import strawberryfields as sf
+import numpy as np
+import pylab as plt
+import matplotlib as mpl
```

Comment:
(minor) Slightly clearer, I think. Just need to change the uses below as well in this case. 🙂 
```suggestion
from matplotlib import colors, colorbar
```
Note that `matplotlib.cm` is the same as `matplotlib.pyplot.cm`, so `plt` can be used there.